.vsconfig

```diff
@@ -55,8 +60,6 @@
     "Component.Microsoft.VisualStudio.LiveShare",
```

Comment:
I get a error about `Component.Microsoft.VisualStudio.LiveShare` not being found. It's not required (and we don't use it), so I think we can remove it.
```suggestion
```